### PR TITLE
COMMERCE-9707 New permission to limit the views of Accounts for Agents based on Account/Channel Relationship

### PR DESCRIPTION
**Pull request by ghost**
https://issues.liferay.com/browse/COMMERCE-9707

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-account-service/src/main/java/com/liferay/commerce/account/internal/util/AccountEntryUtil.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+/**
+ * @author Crescenzo Rega
+ */
+public class AccountEntryUtil {
+
+	public static long getCommerceChannelId(
+			CommerceContext commerceContext, Group group)
+		throws PortalException {
+
+		long commerceChannelId = 0;
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-account-service/src/main/java/com/liferay/commerce/account/internal/util/AccountEntryUtil.java:32]: don't need to declare this variable, just return the value when you get it

Comment (CrescenzoRegaLF) [reply modules/apps/commerce/commerce-account-service/src/main/java/com/liferay/commerce/account/internal/util/AccountEntryUtil.java:32]: I made changes

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce

**Comment by lorenzolapescara** [comment]
QA Passed.

**Comment by lorenzolapescara** [comment]
@CrescenzoRegaLF I've re-sent this with functional tests, can you please close this PR? Thanks!

**Pull request closed without merging**